master/reference/public-cloud/gce.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Calico on Google Cloud
+---
+
+Calico is designed to provide high performance massively scalable virtual
+networking for containers in public cloud environments such as
+Google Cloud.
+
+## Why Use Calico in Google Cloud
+
+#### Network Policy for Containers
+
+While Google Cloud Firewall Rules control network traffic sent to VM instances,
+it does not cover containers running on them. Calico automatically implements
+fine-grain, dynamic network policy for containers.
+
+#### No Overlays
+
+Calico minimizes the need for overlays by assigning each container
+its own IP and allowing containers to talk to one another in a pure IP network.
+
+## How to Run Calico in Google Cloud
+
+To deploy Calico in Google Cloud, ensure that the proper firewall rules
+have been created and that traffic between containers on different hosts is not
+dropped. There are a few different options for doing this depending
+on your deployment.
+
+#### Configure Firewall Rules
+
+Calico requires the following firewall exceptions to enable the necessary
+network connectivity for Calico Google Cloud.
+
+| Description    | Allowed Protocol and Ports |
+|:---------------|:---------------------------|
+| BGP            | tcp:79                     |
```

Comment:
You should note this in the AWS doc as well, but BGP is only required when using BGP networking with ipip (not, for example, when using flannel aws-vpc backend or k8s cloud-provider integration).